include/session/blinding.hpp

```diff
@@ -168,6 +169,19 @@ ustring blind15_sign(ustring_view ed25519_sk, std::string_view server_pk_in, ust
 /// seed + appended pubkey) as with just the 32-byte seed the public key has to be recomputed.
 ustring blind25_sign(ustring_view ed25519_sk, std::string_view server_pk, ustring_view message);
 
+/// Computes a verifiable version-blinded signature that validates with the version-blinded pubkey
+/// that would be returned from blind_version_key_pair.
+///
+/// Takes the Ed25519 secret key (64 bytes, or 32-byte seed), unix timestamp, method, path, and
+/// optional body.
+/// Returns the version-blinded signature.
+ustring blind_version_sign_request(
+        ustring_view ed25519_sk,
+        uint64_t timestamp,
+        ustring_view method,
+        ustring_view path,
```

Comment:
should those be string_view, or we don't care?

Reply:
Good question, and I'm not sure.  I don't know exactly the use case/scenario, but practically speaking they can be whichever is more convenient.

Reply:
`std::string_view`.  Neither of them make any sense containing binary data.

Reply:
(And string_view is always more convenient.  Plus there's the clang+libc++ issues with ustring_views that might necessitate replacing them with spans, which isn't going to be a fun conversion).

Reply:
Changed to `string_view`